concordium-std/tests/derive-deserial-with-state/success-assoc-type.rs

```diff
@@ -0,0 +1,13 @@
+use concordium_std::{DeserialWithState, HasStateApi, ParseResult, Read, StateMap};
```

Comment:
Please document the intent of this test as a module doc comment, i.e., `//!`.

Reply:
Added